src/test/regress/sql/failure_mx_metadata_sync.sql

```diff
@@ -14,7 +14,8 @@ CREATE TABLE t1 (id int PRIMARY KEY);
 SELECT create_distributed_table('t1', 'id');
 INSERT INTO t1 SELECT x FROM generate_series(1,100) AS f(x);
 
--- Initial metadata status
+-- Initially turn metadata sync off
```

Comment:
minor: `Initially turn metadata sync off because we'll ingest errors to start/stop metadata sync operations`